### PR TITLE
Post card and Auth Functionality

### PR DESCRIPTION
**Pull request by sashaboi**
### Check : AUTH ONLY
If you wish to login as one of the defualt users, you need to signup as them for now. will update the exisitng users in later prs.

Was able to make auth functionality and design a card which could handle all the posts. 
Still lots to do.

### CODE REVIEW

File: src/components/PostCard/postcard.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.post-card-parent {
+  color: var(--font-color);
+  background-color: var(--bg-color);
+  padding: 0.5rem 0.5rem;
+  border-bottom: 1px solid var(--secondary-color);
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
```
Comment (TejasShekar) [inline src/components/PostCard/postcard.css:8]: 10px

File: src/pages/Auth/Login.jsx [hunk 2]
```diff
@@ -12,16 +34,52 @@ const Login = () => {
...
+                setEmail('bill.gates@gmail.com');
+                setPassword('adarshBalika123');
+              }}
+              className="secondary-btn"
+            >
+              Login as Bill Gates
+            </button>
+            <button
+              onClick={() => {
+                setEmail('elon.musk@gmail.com');
```
Comment (TejasShekar) [inline src/pages/Auth/Login.jsx:67]: Cannot login as Elon Musk

File: src/components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
+import { Link, useNavigate } from 'react-router-dom';
 import './navbar.css';
 export const Navbar = () => {
+  const navigate = useNavigate();
+  const LogoutHandler = () => {
+    localStorage.clear();
+    navigate('/');
+  };
+  const token = localStorage.getItem('token');
+  console.log(token);
```
Comment (TejasShekar) [inline src/components/Navbar/Navbar.jsx:11]: remove console.log if not needed

File: src/components/PostCard/PostCard.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import './postcard.css';
+
+import { FaRegComment, FaRegHeart } from 'react-icons/fa';
+
+export const PostCard = () => {
+  return (
+    <div className="post-card-parent">
+      <div className="post-top-content">
+        <div className="profile-data">
+          <img src="" alt="profile pic" />
```
Comment (TejasShekar) [inline src/components/PostCard/PostCard.jsx:11]: add user image/picsum for now

**Review by TejasShekar — APPROVED** [review]
Auth working good. Just a few suggestions mentioned.

File: src/components/PostCard/postcard.css (hunk 1 above)
Comment (sashaboi) [reply src/components/PostCard/postcard.css:8]: will add.

File: src/pages/Auth/Login.jsx (hunk 2 above)
Comment (sashaboi) [reply src/pages/Auth/Login.jsx:67]: fixed.

File: src/components/PostCard/PostCard.jsx (hunk 4 above)
Comment (sashaboi) [reply src/components/PostCard/PostCard.jsx:11]: agreed. adding in next version.

**Pull request merged**